rekognition_tutorial/short_sa_rekognition_tutorial.ipynb

```diff
@@ -0,0 +1,1222 @@
+    "        project_version_running_waiter.wait(ProjectArn=project_arn, VersionNames=[version_name])\n",
+    "\n",
+    "        #Get the running status\n",
+    "        describe_response=client.describe_project_versions(ProjectArn=project_arn,\n",
+    "            VersionNames=[version_name])\n",
+    "        for model in describe_response['ProjectVersionDescriptions']:\n",
+    "            print(\"Status: \" + model['Status'])\n",
+    "            print(\"Message: \" + model['StatusMessage']) \n",
+    "    except Exception as e:\n",
+    "        print(e)\n",
+    "        \n",
+    "    print('Done...')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70110780",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start_model(rek_client,\n",
+    "            project_arn, \n",
+    "            model_arn, \n",
+    "            version_name, \n",
+    "            min_inference_units = 1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a2f699d",
+   "metadata": {},
+   "source": [
+    "After the cell above is finished we could see that model status on Rekognition page is now \"Running\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ffe04696",
+   "metadata": {},
+   "source": [
+    "![](./tutorial_images/model_is_running.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c3525a10",
+   "metadata": {},
+   "source": [
+    "## Prediction test\n",
+    "<a id='predict'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35f21b17",
+   "metadata": {},
+   "source": [
+    "Now the model is ready to run and we can get prediction for any unlabeled image we have.\n",
+    "We have to provide a path to S3 bucket and folder where the image is located."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1468d369",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def show_custom_labels(client, model, bucket, photo, min_confidence):\n",
+    "    response = client.detect_custom_labels(Image = {'S3Object': {'Bucket': bucket, \n",
+    "                                                                 'Name': photo}},\n",
+    "                                           MinConfidence = min_confidence,\n",
+    "                                           ProjectVersionArn = model)\n",
+    "    return response['CustomLabels']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f5be1fd6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename = 'FILE TO TEST'\n",
+    "photo = f'{BUCKET_FOLDER_PATH}/unlabeled/{filename}'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "68398604",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "min_confidence = 10 ## minimum confindence model should have for the label to detect it\n",
+    "\n",
+    "%time labels = show_custom_labels(rek_client, model_arn, bucket_name, photo, min_confidence)\n",
```

Comment:
Why do you need %time? Is it important for the tutorial?

Reply:
+